test/run-test.sh

```diff
@@ -22,7 +19,15 @@ fi
 npx cdk --no-version-reporting -a ${cdk_app} synth > ${actual}
 
 if [ "${1:-}" == "update" ]; then
+  hash="$(cat ${actual} | md5sum | cut -c1-6 | xargs)"
+  export TEST_STACK_NAME="delivlib-test-${hash}"
   npx cdk --no-version-reporting -a ${cdk_app} deploy ${2:-} ${3:-} ${4:-}
+  echo "Stack deployed, now, go to the console and wait for the pipeline to fully stabalize"
+  echo "Press ENTER to confirm that pipeline is green"
+  read
+  echo "Okay, now go to CFN console and delete the test stack ${TEST_STACK_NAME}"
```

Comment:
Would be sweet if we could actually... you know... go and check that the pipeline is, indeed, green. We can have this a TODO / separate issue, though.

Reply:
Totally could. Totally in a separate issue.

Reply:
#49